### PR TITLE
create User model

### PR DESCRIPTION
**Pull request by sam96B**

### CODE REVIEW

File: src/models/UserModel.ts [hunk 1]
```diff
@@ -45,19 +27,17 @@ User.init(
...
+      allowNull: false
+    },
+    role: {
+      type: DataTypes.STRING(128),
+      allowNull: false
+    }
   },
   {
-    tableName: 'users',
+    modelName: 'User',
```
Comment (AhmedSafi97) [inline src/models/UserModel.ts:40]: table naming should be plural all lowercase

File: src/models/UserModel.ts [hunk 2]
```diff
@@ -1,30 +1,12 @@
...
-import { Book } from './BookModel'
+import { DataTypes, Model } from 'sequelize'
 
-export class User extends Model<
-  InferAttributes<User, { omit: 'books' }>,
-  InferCreationAttributes<User, { omit: 'books' }>
-> {
-  declare id: CreationOptional<number>
+import { sequelize } from '../db/connection'
+export class User extends Model {
```
Comment (AhmedSafi97) [inline src/models/UserModel.ts:4]: leave empty lines between your code blocks

File: src/models/UserModel.ts [hunk 3]
```diff
@@ -1,30 +1,12 @@
...
+  declare id: number
```
Comment (AhmedSafi97) [inline src/models/UserModel.ts:5]: this should be creation optional

**Review by AhmedSafi97 — CHANGES_REQUESTED** [review]

File: src/models/UserModel.ts [hunk 4]
```diff
@@ -1,30 +1,12 @@
...
   declare email: string
-
-  declare createdAt: CreationOptional<Date>
-  declare updatedAt: CreationOptional<Date>
-
-  declare books?: NonAttribute<Book[]>
-  declare static associations: {
-    books: Association<User, Book>
-  }
+  declare role: string
```
Comment (AhmedSafi97) [inline src/models/UserModel.ts:9]: Look at this comment [here](https://github.com/CA-G12/wms-solution-inc/pull/4#discussion_r996421637)

we will need the same for the `role` ['admin', 'customer']

**Review by AhmedSafi97 — DISMISSED** [review]
LGTM 

But we still need a slight change, I've left a comment below :pray:

**Review by Rabee96 — APPROVED** [review]

**Review by ShbairAli — DISMISSED** [review]

**Review by AhmedSafi97 — APPROVED** [review]

**Pull request merged**